wire-ios-sync-engine/Source/SessionManager/BackendEnvironmentProvider+Reachability.swift

```diff
@@ -83,6 +83,7 @@ final class ReachabilityWrapper: NSObject, ReachabilityProvider, TearDownCapable
         didSet {
             if safeReachability == nil && enabled {
                 safeReachability = reachabilityClosure()
+                NotificationCenter.default.post(name: NSNotification.Name(rawValue: ZMTransportSessionReachabilityIsEnabled), object: nil)
```

Comment:
```suggestion
                NotificationCenter.default.post(name: NSNotification.Name(rawValue: ZMTransportSessionReachabilityIsEnabled), object: self)
```
We could limit to check only the ZMTransportSession that depends on this ReachabilityWrapper renews its observations